### DISCUSSION TITLE
How do I use backup slides (at the end of a revealjs presentation) without them showing in the TOC?

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#5962] [Q&A] [answered]
### Description

It's a common use case (for me, anyway, doing presentations for 20+ years) to put _backup slides_ at the end of a presentation. They're useful in case of questions, etc. and I don't include them in the main part because of time limitations. 

The problem with this pattern is that backup slides show up in the Table of Contents slide (a very useful feature). I have tried putting `{.unnumbered}` on the backup slides, but it seems to have no impact on their appearance in the TOC. Is there another directive to keep them out of the table of contents? 

```markdown
---
format: 
  revealjs:
    toc: true
    toc-depth: 1
    toc-title: Overview
---

# Introduction

Topic

# Results

blah

# Conclusion

Etc.

<!-- backup slides I don't want to show in the TOC -->

# Weird situation 1 {.unnumbered}

Lore ipsum

# Weird situation 2 {.unnumbered}

blah dee blah

## special case {.unnumbered}

(vertical slide) ....

```

**Comment by mcanouil** [comment]
See:
- #4798
- #5566

**Comment by fuhrmanator** [comment, marked as answer]
The solution is with both `{.unnumbered .unlisted}` according to the HTML docs for Table of Contents at https://quarto.org/docs/output-formats/html-basics.html#table-of-contents

@mcanouil cited ways to use filters to make it work for everything in a section, which is great, too. The lua filter in one of those discussions could be modified in accordance.

But I'm happy to live with tagging all my backup slides with these two classes.

**Reply by mcanouil** [reply]
I did not know about `.unlisted` 😅
